code/datums/components/foldable.dm

```diff
@@ -2,19 +2,23 @@
 	var/obj/item/objBriefcase/the_briefcase
 	var/original_name
 	var/original_desc
+	var/briefcase_path
 	var/change_name = 1
 
 /datum/component/foldable/keep_name
 	change_name = 0
 
-/datum/component/foldable/Initialize()
+/datum/component/foldable/Initialize(var/briefcase_path = /obj/item/objBriefcase)
 	if(!istype(parent, /atom/movable))
 		return COMPONENT_INCOMPATIBLE
+	if(!ispath(briefcase_path, /obj/item.objBriefcase))
```

Comment:
... what? Does this even work?

Should be `/obj/item/objBriefcase`

Reply:
it did! but still changed it